src/node/coinstats.h

```diff
@@ -76,6 +76,8 @@ std::optional<CCoinsStats> GetUTXOStats(CCoinsView* view, node::BlockManager& bl
 uint64_t GetBogoSize(const CScript& script_pub_key);
 
 CDataStream TxOutSer(const COutPoint& outpoint, const Coin& coin);
+
+CCoinsStats MakeCoinStatsPrefilledWithBlockIndexInfo(const CBlockIndex* pindex);
```

Comment:
Why not make this a ctor which takes a `CBlockIndex*` and prefills itself?

Reply:
Agree, I was going to leave a nit that this is a bit verbose for my taste ;)

Reply:
Fixed as of push 8b853034a13f49b85cdf4168d9d49bd297f5b8ce